src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -56,8 +57,9 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Set<Appointment> appointmentList = ParserUtil.parseAppointments(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, nric, address, dob, gender, tagList);
+        Person person = new Person(name, phone, email, nric, address, dob, gender, tagList, appointmentList);
```

Comment:
For `appointmentList`,  are you planning to make it such that the user can choose to also input the patient's appt using the existing `t/` tag prefix when adding the patient and it will get parsed by `parseappointment` and only added if it is a valid appointment?

If not, should we set  `appointmentList` to be an empty set first? So that the `Person` object will be initialise with no appts.

Reply:
Hm I think it's a good idea, but I think the problem is each appt takes in 3 arguments, so I think it might be too hassle for when adding new patient. 

For now I will stick to making it default to empty list first. Maybe it be another feature.